torch/quantization/utils.py

```diff
@@ -1,9 +1,87 @@
 """
 Utils shared by different modes of quantization (eager/graph)
 """
+import torch
+from .quant_type import QuantType, quant_type_to_str
 
 def get_combined_dict(default_dict, additional_dict):
```

Comment:
Does this need to be a public API?

Reply:
this is not exposed in torch.quantization, not sure what you mean by public api